packages/ag-charts-community/src/scene/layerNew.ts

```diff
@@ -0,0 +1,163 @@
+import { BBox } from './bbox';
+import type { HdpiCanvas } from './canvas/hdpiCanvas';
+import { nodeCount } from './debug.util';
+import { Group } from './group';
+import type { LayersManager, ZIndexSubOrder } from './layersManager';
+import { type ChildNodeCounts, RedrawType, type RenderContext } from './node';
+import { Translatable } from './transformable';
+
+export class LayerNew extends Group {
```

Comment:
Can we roll this back into `Layer` please, with the new behaviour controlled by a constructor option, it makes it much easier to reason about the difference between this and the existing `Layer` implementation?

Reply:
Updated